pxtrunner/runner.ts

```diff
@@ -24,6 +24,7 @@ namespace pxt.runner {
         additionalQueryParameters?: string;
         debug?: boolean;
         mpRole?: "server" | "client";
+        sendBuilt?: boolean;
```

Comment:
```suggestion
```

these ones aren't used, change in run.html should be fine on it's own~

Reply:
I was getting in-editor and runtime errors when I didn't have the other locations were not defined. Are they something I can ignore? I also figured I should add them since the edits you made recently for multiplayer also included these types of changes.

Reply:
Oh, I think I understand why. Is it because the existence of the query parameter doesn't really effect the runtime of the simulator at all? It effects `simulateAsync` and the simOptions paramenter, but `builtJsInfo` is already included in that simOptions, so it doesn't need to be in the larger objects themselves. Is that about right?

Reply:
the reason we shouldn't need them is because the sendBuilt portion is handled solely in run.html -- that is, the flow is something like

```
kiosk -> run.html -> pxtrunner ->  pxtcompiler (building compile info) and pxtsim (simdriver)
```

where kiosk embeds run.html, which calls into pxtrunner, which calls into pxtcompiler to build the game and pxtsim to create a simulator / run the game. You have to add more fields depending on how deep the flag has to effect, but in this case pxtrunner already gives back the buildjsinfo due to a change I had previously made for the arcade share page, so the changes only need to exist in run.html to send that game back up for kiosk to read, and no types have to be added for pxtrunner / pxtsim

Reply:
Okay, yes, that makes sense. Thank you for the clarification!